habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -161,7 +163,15 @@ def reconfigure(self, config: Config):
         self.ep_info = ep_info
         self._try_acquire_context()
 
-        if self.prev_scene_id != ep_info["scene_id"]:
+        self._targets = {}
+        for target_handle, transform in self.ep_info["targets"].items():
+            self._targets[target_handle] = mn.Matrix4(
```

Comment:
Do we have any assert here that transform[i,j] is of len(4)? Maybe consider adding it.

Reply:
Good catch, the error would have been very obscure to a new user. I added some asserts with a message